src/Widgets/Status.vala

```diff
@@ -99,6 +104,116 @@ public class Tootle.Widgets.Status : ListBoxRow {
 			return status.formal.account.avatar;
 		}
 	}
+	public void get_poll(){
```

Comment:
`Widgets/Status.vala` is already bloated as it is. The voting box should be a separate `Widgets/VoteBox.vala` file that handles the voting logic as well as populating the voting options.

It should also use a Glade ui template, just like any other view component in Tootle.